src/client/App.tsx

```diff
@@ -75,7 +75,7 @@ const useMe = () => {
   const path = '/user-api/users/me'
   return useQuery<Me, AxiosError>(
     path,
-    async () => (await api.get<Me>(path)).data,
+    async () => (await api.get<Me>(path))?.data,
```

Comment:
I think this should solve 
<img width="1134" alt="Screenshot 2025-07-09 at 12 06 54 PM" src="https://github.com/user-attachments/assets/389e33da-a4b5-4274-9da6-60fb740b2ab8" />